packages/babel-plugin-syntax-import-meta/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@babel/plugin-syntax-import-meta",
+  "version": "7.0.0-beta.34",
+  "description": "Allow parsing of import.meta",
+  "repository": "https://github.com/babel/babel/tree/master/packages/babel-plugin-syntax-import-meta",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "keywords": [
+    "babel-plugin"
+  ],
+  "peerDependencies": {
+    "@babel/core": "7.0.0-beta.34"
+  },
+  "devDependencies": {
+    "@babel/core": "7.0.0-beta.34"
```

Comment:
This devDependency isn't needed.

Reply:
@nicolo-ribaudo This should be consistent with out other modules, so having it is the way to go.

Reply:
@loganfsmyth whats the point of having peerDep here then?

Reply:
@andarist A peer just declares a peer without installing anything, so the dev dependency is needed for Lerna to add the symlinks to `@babel/core`. See my other comment though. That peerdep is supposed to be a general range, so it wouldn't normally be a repeat of the dev dependency.

Reply:
@andarist A peer just declares a peer without installing anything, so the dev dependency is needed for Lerna to add the symlinks to `@babel/core`. See my other comment though. That peerdep is supposed to be a general range, so it wouldn't normally be a repeat of the dev dependency.